rd_ui/app/scripts/controllers/dashboard.js

```diff
@@ -125,6 +125,12 @@
   };
 
   var WidgetCtrl = function($scope, $location, Events, Query) {
+    $scope.disableWidget = {'visibility': 'visible'};
+    $scope.checkWidth = function (width) {
+     if (width == 0)
+        $scope.disableWidget = {'visibility': 'hidden'};
```

Comment:
Why aren't you setting this in the controller, instead of in the `checkWidth` function?